src/attendee/tests.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.conf import settings
+from registry.helper import reg
+from attendee.models import Attendee
+from core.models import Token
+
+
+@pytest.mark.parametrize('attendee_token,status,num_channel', [
+    ('123', 400, 0),
+    ('1234', 200, 0),
+    ('1234', 200, 1),
+    ('1234', 200, 2),
```

Comment:
**nitpick:** it would be more meaningful for the real scenario if passing the fake data as the value of a token (i.e. `7521d8fa2ede2ecd4a77b5dgca1e921c`)